sdks/python/apache_beam/io/gcp/bigtableio.py

```diff
@@ -106,10 +121,17 @@ def process(self, row):
     #                     'field1',
     #                     'value1',
     #                     timestamp=datetime.datetime.now())
-    self.batcher.mutate(row)
+    try:
```

Comment:
We need to instrument the underlying code which calls the BigTable API. Not the batcher which wraps around it. So we can properly count all the request made to bigquery. So I think this needs to go elsewhere.

If its not obvious where this should go, ask in the beam dev list, where this might be.
We want to capture the underlying API calls, and their http or grpc errors.
This may require modifying the Bigtable library